core-services/prow/02_config/quay/quay-operator/_pluginconfig.yaml

```diff
@@ -0,0 +1,38 @@
+external_plugins:
+  quay/quay-operator:
+  - endpoint: http://refresh
+    events:
+    - issue_comment
+    name: refresh
+  - endpoint: http://cherrypick
+    events:
+    - issue_comment
+    - pull_request
+    name: cherrypick
+  - endpoint: http://needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+    name: needs-rebase
+plugins:
+  quay/quay-operator:
+    plugins:
+    - bugzilla
```

Comment:
we use JIRA to track bugs, not BZ - do we still need it?

Reply:
It doesn't matter, you won't see it unless you create a PR with a title like `Bug XXXXX: ...`. But let's remove it so it doesn't confuse others.